packages/dates/src/utilities/formatDate.ts

```diff
@@ -9,11 +9,32 @@ interface FormatDateOptions extends Intl.DateTimeFormatOptions {
   hourCycle?: string;
 }
 
+interface ResolvedDateOptions extends Intl.ResolvedDateTimeFormatOptions {
```

Comment:
We should extend this globally like we did for `Intl.DateTimeFormatOptions`

Reply:
@carysmills do you remember why these were moved back into this file?